### PR TITLE
Initial skeleton implementation of chs-delta-api

### PR DESCRIPTION
**Pull request by mreis-ch**
The skeleton API consists of all the required files for Mesos/Marathon, e.g. Makefile, start script, routes.yaml, etc.
A healthcheck endpoint has also been added

DSND-15

### CODE REVIEW

**Review by amoriartyCH — APPROVED** [review]
Nice work!

File: go.mod [hunk 1]
```diff
@@ -0,0 +1,10 @@
+module github.com/companieshouse/chs-delta-api
+
+go 1.12
+
+require (
+	github.com/companieshouse/chs.go v1.2.1
```
Comment (divya051988) [inline go.mod:6]: Please use the latest version of chs.go

File: go.mod [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+	github.com/companieshouse/gofigure v0.1.4
+	github.com/gorilla/mux v1.7.3
```
Comment (divya051988) [inline go.mod:8]: Latest version please

File: go.mod [hunk 3]
```diff
@@ -0,0 +1,10 @@
...
+	github.com/smartystreets/goconvey v0.0.0-20190731233626-505e41936337
```
Comment (divya051988) [inline go.mod:9]: Same as above

**Pull request merged**